shoop/admin/views/dashboard.py

```diff
@@ -17,6 +18,7 @@ class DashboardView(TemplateView):
 
     def get_context_data(self, **kwargs):
         context = super(DashboardView, self).get_context_data(**kwargs)
+        context["version"] = _version.__version__
```

Comment:
You shouldn't import or use a private submodule from another python package, i.e. one that starts with an underscore (in this case `_version`). Please just `import shoop` and use `shoop.__version__`.